### PR TITLE
[tfldump] Enable FAKE_QUANT

### PR DESCRIPTION
**Pull request by seanshpark**
This will enable tflite file dump of FAKE_QUANT Op.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #5610, from draft #5625

File: compiler/tfldump/src/OpPrinter.cpp [hunk 1]
```diff
@@ -305,6 +305,24 @@ class DepthwiseConv2DPrinter : public OpPrinter
...
+public:
+  void options(const tflite::Operator *op, std::ostream &os) const override
+  {
+    if (auto *params = op->builtin_options_as_FakeQuantOptions())
+    {
+      os << "    ";
+      os << "Min(" << params->min() << ") ";
+      os << "Max(" << params->max() << ") ";
+      os << "NumBits(" << params->num_bits() << ") ";
+      os << std::boolalpha;
```
Comment (llFreetimell) [inline compiler/tfldump/src/OpPrinter.cpp:319]: AFAIK, unless `std::noboolalpha` should be called at the end of this function, other codes can be affected.

Comment (seanshpark) [reply compiler/tfldump/src/OpPrinter.cpp:319]: This is what I get
```
O(0:0) FAKE_QUANT
    Min(0) Max(1) NumBits(8) NarrowRange(false)
    I T(0:0) ifm
    O T(0:1) ofm
```
You mean we should turn off with `noboolalpha`?
There already exists several uses if `boolalpha` and seems OK...

Comment (seanshpark) [reply compiler/tfldump/src/OpPrinter.cpp:319]: Done some test and it would be safe to put `noboolalpha` after boolean dump.

**Review by llFreetimell — APPROVED** [review]
LGTM, thanks!

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**